app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js

```diff
@@ -1,22 +1,70 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import { capitalize } from 'lodash';
 import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/react-hooks';
+import { decodeQueryParams } from 'use-query-params';
 
 import Tab from '../shared/tabs/Tab';
 import Tabs from '../shared/tabs/Tabs';
 import TabBody from '../shared/tabs/TabBody';
 import ContextualNavbar from '../shared/ContextualNavbar';
+import ResultsPagingBar from '../shared/ResultsPagingBar';
 import DigitalObjectSummary from './DigitalObjectSummary';
-import { encodeAndStringifySearch } from '../../utils/encodeAndStringifySearch';
+import { getDigitalObjectIDsQuery } from '../../graphql/digitalObjects';
+import { queryParamsConfig, encodeAndStringifySearch } from '../../utils/encodeAndStringifySearch';
+import GraphQLErrors from '../shared/GraphQLErrors';
+
 
 function DigitalObjectInterface(props) {
   const { digitalObject, children } = props;
   const { id, title, digitalObjectType } = digitalObject;
-  const latestSearchQueryString = sessionStorage.getItem('searchQueryParams');
+  const latestSearchQueryString = window.sessionStorage.getItem('searchQueryParams');
+  const history = useHistory();
+  const offset = window.sessionStorage.getItem('offset');
+  const resultIndex = window.sessionStorage.getItem('resultIndex');
+  const limit = 3;
+
+  const queryParams = {
+    q: undefined,
+    pageNumber: undefined,
+    perPage: undefined,
+    filters: undefined,
+    orderBy: undefined,
+    ...JSON.parse(latestSearchQueryString),
+  };
+
+  const {
+    q, filters, orderBy,
+  } = decodeQueryParams(queryParamsConfig, queryParams);
```

Comment:
```suggestion
  } = decodeQueryParams(queryParamsConfig, queryParams);
  
  const [orderField, orderDirection] = orderBy.split(' ');
```

Maybe good to double check my syntax locally, but I think the above destructuring assignment should work.  This suggestion is related to another suggestion below.